package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "web-push",
-  "version": "3.4.5",
+  "version": "3.4.6",
```

Comment:
This should be done separately, can you revert it from the PR?

Reply:
Ok